<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ruledialog_m</class>
 <widget class="QDialog" name="ruledialog_m">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>源端口：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_time">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_src_port">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>any</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>协议：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>动作：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_src_ip">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>any</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dst_ip">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>any</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>目的地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>时间：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_log">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_protocol">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>日志：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_action">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>源地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>目的端口：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dst_port">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>any</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
